resources/js/Components/EventPopUp.vue

```diff
@@ -0,0 +1,55 @@
+<script setup lang="ts">
+import { Link as InertiaLink } from '@inertiajs/vue3'
+import { formatDate, formatTime } from '@/utilities/formatDate'
+import BaseButton from '@/Components/BaseButton.vue'
+import type { EventMarker } from '@/types/events'
+
+const props = defineProps<{ event: EventMarker }>()
```

Comment:
You don't need to use the props constant when accessing props in `<script setup>` in **Vue 3**. Props are available directly by their name, so you can use `event` instead of `props.event`. Check other components